jekyll/_cci2/api-intro.md

```diff
@@ -0,0 +1,122 @@
+
+The current categories of the API v2 endpoints are:
+
+- Authentication
+- Pipeline
+- Workflows
+- User (Preview)
+- Project (Preview)
+- Job (Preview)
+
+
+**Note:** Portions of the CircleCI API v2 remain under "Preview". Preview endpoints are not yet fully supported or considered generally available. Breaking changes to API v2 Preview endpoints are planned in advance and are announced in the [API v2 breaking changes log](https://github.com/CircleCI-Public/api-preview-docs/blob/master/docs/breaking.md).
+
+## Getting Started with the API v2
+
+The CircleCI API v2 is backwards-compatible with previous API versions in the way it identifies your projects using repository name. For instance, if you want to pull information from CircleCI about the GitHub repository https://github.com/CircleCI-Public/circleci-cli you can refer to that in the CircleCI API as `gh/CircleCI-Public/circleci-cli`, which is a "triplet" of the project type, the name of your "organization", and the name of the repository. For the project type you can use `github` or `bitbucket` as well as the shorter forms `gh` or `bb`, which are now supported in API v2. The `organization` is your username or organization name in your version control system.
+
+With API v2, CircleCI is introducing a string representation of the triplet called the `project_slug`, which takes the following form:
+
+`<project_type>/<org_name>/<repo_name>`
+
+The `project_slug` is included in the payload when pulling information about a project as well as when looking up a pipeline or workflow by ID. The `project_slug` can then be used to get information about the project. It is possible in the future the shape of a `project_slug` may change, but in all cases it would be usable as a human-readable identifier for a given project.
+
+### Authentication
+
+The CircleCI API v2 enables users to be authenticated by simply sending your API token as the username of the HTTP request. For example, if you have set `CIRCLECI_TOKEN` in your shell's environment, then you could then use `curl` with that token like the example shown below:
+
+`curl -user ${CIRCLECI_TOKEN} https://circleci.com/api/v2/me`
+
+### Pipelines
+
+The CircleCI API v2 includes the use of pipelines to assist you in triggering workflows from the CircleCI API. By enabling pipelines, you can take advantage of specific API use cases such as:
+
+- Jobs named `build` will be wrapped in a workflows stanza by the processor.
+- Projects for which auto-cancel is enabled in the **Advanced Settings** will have workflows on non-default branches cancelled when a newer build is triggered on that same branch.
+
+For more detailed information about pipelines and how you can use them in your workflows and builds, please see the [Build Processing](https://circleci.com/docs/2.0/build-processing/)page.
```

Comment:
I think the Build Processing page is now titled Enabling Pipelines .... Maybe here refer to it as the Pipelines page here?